components/card/card.css

```diff
@@ -0,0 +1,113 @@
+.card {
+    padding: 1rem;
+    width: 15.5rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.card-border-radius {
+    border-radius: 0.625rem;
+}
+
+.card-vertical {
+    padding: 0;
+    width: 17.2rem;
+}
+
+.card-horizontal {
+    padding: 0;
+    width: 30rem;
+    flex-direction: row;
+}
+
+.card-margin-bottom {
+    margin: 0 0 0.5rem 0;
+}
```

Comment:
just a suggestion you can just use margin-bottom.

Reply:
actually, I need that format of margin bottom. so I used it.